app/services/irs_attempts_api/tracker.rb

```diff
@@ -18,8 +18,6 @@ def initialize(session_id:, request:, user:, sp:, cookie_device_uuid:,
     def track_event(event_type, metadata = {})
       return unless enabled?
```

Comment:
I think this entire method can be empty! We can remove all the stuff about being enabled, we can probably go up and yank the `session_id` that we get from the OIDC API too